src/lib/crypto/kimchi_bindings/wasm/src/pasta_fq_plonk_index.rs

```diff
@@ -37,6 +37,8 @@ pub fn caml_pasta_fq_plonk_index_create(
     prev_challenges: i32,
     srs: &WasmSrs,
 ) -> Result<WasmPastaFqPlonkIndex, JsValue> {
+    console_error_panic_hook::set_once();
```

Comment:
instead should we just set it in an init function as indicated here? https://github.com/rustwasm/console_error_panic_hook#usage

Reply:
yeah sorry, I didn't now how to make an init function that runs at startup, but that would be even cleaner